src/GovUk.Education.ExploreEducationStatistics.Data.Processor/FailedImportsHandler.cs

```diff
@@ -23,6 +25,8 @@ public static void CheckIncompleteImports()
 
             aQueue.CreateIfNotExists();
             table.CreateIfNotExists();
+            aQueue.Clear();
+            //pQueue.Clear();
```

Comment:
I think it would be good to rename these queues as it's no obvious what they are. How come pQueue is not cleared anymore?

Reply:
I've cleared the pending queue but there is a small issue which I'm going to address in a new ticket EES-1425.